packages/dds/tree/src/test/shared-tree/treeApi.spec.ts

```diff
@@ -161,4 +173,395 @@ describe("treeApi", () => {
 
 		// TODO: When SchematizingSimpleTreeView supports forking, add test coverage to ensure that transactions work properly on forks
 	});
+
+	describe("on", () => {
```

Comment:
This new suite is identical to the one in `src/test/simple-tree/treeApi.spec.ts`. Not sure if we have a particular take on "don't duplicate because one just exposes the other" VS "let's make sure both are tested so we don't accidentally break shared-tree while somehow managing to keep simple-tree working".